-tags not working, along with zig failures

### Description

Have an application I've been able to build for around a year with fyne-cross, without issue. Today I upgraded to the latest `1.4.0` and am now running into multiple issues.

The project has a `configs/` directory that houses numerous `.go` files, with build-tags within each file, used for setting app configuration values. Issuing a `go build -tags dev .` runs just like you'd expect. When running with `fyne-cross linux -tags dev .` it fails to honor the file with the matching tag.

```
config/dev.go:51:14: undefined user
....
```
_(every value set within the config file, for the matching tag, yields an undefined error)_

If I remove the build tag from the specific configuration file I want to use—_forcing the config without tags_—I then end up with a `zig` linking failure:

```
# fyne-cross linux -tags dev .
...
/usr/local/go/pkg/tool/linux_amd64/link: running zig failed: exit status 1
ld.lld: error: /tmp/go-link-3345816007/000000.o: unknown file type
...
```


### To Reproduce:

1. Create dummy hello-world app
    ```
    % mkdir helloworld
    % cd helloworld
    % go mod init helloworld
    % go mod tidy ← after creating the below files
    ```
    - Create `main.go`
    ```
    package main

    import (
        "fmt"

        "fyne.io/fyne/v2/app"
        "fyne.io/fyne/v2/widget"
    )

    var Name = name

    func main() {
        a := app.New()
        w := a.NewWindow("Hello World")

        w.SetContent(widget.NewLabel(fmt.Sprintf("Hello %s", Name)))
        w.ShowAndRun()
    }
    ```
    - Create `vanessa.go`
    ```
    //go:build vanessa
    // +build vanessa

    package main

    var name = "Vanessa"
    ```
2. Build/Run _(works as expected)_
   ```
   go build -tags vanessa .
   /helloworld`
   ```
3. Fail w/fyne-cross
   ```
    fyne-cross linux -tags vanessa .
    [i] Target: linux/amd64
    [i] Cleaning target directories...
    [✓] "bin" dir cleaned: /app/fyne-cross/bin/linux-amd64
    [✓] "dist" dir cleaned: /app/fyne-cross/dist/linux-amd64
    [✓] "temp" dir cleaned: /app/fyne-cross/tmp/linux-amd64
    [i] Checking for go.mod: /path/to/helloworld/go.mod
    [✓] go.mod found
    [i] Packaging app...
    # helloworld
    ./main.go:10:12: undefined: name

    error building application: exit status 1
    [✗] could not package the Fyne app: could not package the Fyne app: exit status 1
   ```

### Device and debug info

 - **OS:** Linux
 - **Version:** 6.1.25-1-MANJARO
 - **Go version:** go version go1.20.3 linux/amd64
 - **fyne-cross version:** fyne-cross version v1.4.0
 - **Fyne version:** 2.3.4

--- a/internal/command/command.go
+++ b/internal/command/command.go
@@ -191,7 +191,7 @@ func fyneCommand(binary, command, icon string, ctx Context, image containerImage
 	// add tags to command, if any
 	tags := image.Tags()
 	if len(tags) > 0 {
-		args = append(args, "-tags", fmt.Sprintf("%s", strings.Join(tags, ",")))
+		args = append(args, "-tags", strings.Join(tags, ","))
 	}
 
 	if ctx.Metadata != nil {

--- a/internal/command/command.go
+++ b/internal/command/command.go
@@ -191,7 +191,7 @@ func fyneCommand(binary, command, icon string, ctx Context, image containerImage
 	// add tags to command, if any
 	tags := image.Tags()
 	if len(tags) > 0 {
-		args = append(args, "-tags", fmt.Sprintf("%q", strings.Join(tags, ",")))
+		args = append(args, "-tags", fmt.Sprintf("%s", strings.Join(tags, ",")))
 	}
 
 	if ctx.Metadata != nil {

